File: tests/conformance/fixtures/go/26-mandate-human.go
package main

import sykli "github.com/false-systems/sykli/sdk/go"

func main() {
	p := sykli.New()
	p.Task("docs").
		Run("make docs").
		Actor(sykli.ActorKindHuman, "maintainer").
		Mandate(sykli.NewMandate(
			[]string{"docs/**", "README.md"},
			sykli.DiffLines(200),
			sykli.WallClockMS(900000),
			sykli.Network(false),
		))
	p.Emit()
}

File: tests/conformance/fixtures/typescript/26-mandate-human.ts
import { Pipeline } from '../../../../sdk/typescript/src/index';

const p = new Pipeline();
p.task('docs')
  .run('make docs')
  .actor({ kind: 'human', id: 'maintainer' })
  .mandate({
    scope: ['docs/**', 'README.md'],
    budget: { diff_lines: 200, wall_clock_ms: 900000 },
    capabilities: { network: false },
  });
p.emit();
